src/rodeo/goat_herd.rs

```diff
@@ -199,4 +215,17 @@ async fn test_purls_and_merge() {
     "Expecting some history, got {:?}",
     history
   );
+
+  let tags = herd2
+    .item_for_identifier("tags")
+    .expect("Should get tags")
+    .expect("Should get tags from option");
+  let tagged: Vec<String> = tags
+    .connections
+    .iter()
+    .filter(|conn| conn.0.is_tag_to())
+    .map(|conn| conn.1.clone())
+    .collect();
+
+  assert_eq!(tagged.len(), 2, "Expecting 2 tags, got {:?}", tagged);
```

Comment:
Should you assert that the tags are different? (In general, I like seeing tests that show the actual data, so asserting specific values is a good strategy too)

Reply:
Thank you! Added.